lib/stream/xlsx/workbook-reader.js

```diff
@@ -20,11 +20,12 @@ class WorkbookReader extends EventEmitter {
 
     this.input = input;
 
-    this.options = options || {
-      hyperlinks: 'cache',
-      sharedStrings: 'cache',
-      worksheets: 'emit',
-    };
+    this.options = options || {};
```

Comment:
I think this default options handling method is ok, I've just wanted to share with you about what I think is perfect:

```js
const defaults = Object.freeze({a:2,b:3,c:4}); //may be const for class

function someFunc(options){
  options = Object.freeze(Object.assign({},defaults,options))
  // ...
  console.log(options); //options is immutable
};

someFunc({a:1,c:5});
someFunc();
someFunc("asd");
someFunc(123);
someFunc([123]);
someFunc(null);
someFunc({});
```

![image](https://user-images.githubusercontent.com/2893099/78617138-afda1300-7876-11ea-833c-12edb70968c2.png)

it's only for info, I don't need changes here 😄

Reply:
let's go with 
```js
    this.options = {
      worksheets: 'emit',
      sharedStrings: 'cache',
      hyperlinks: 'ignore',
      styles: 'ignore',
      entries: 'ignore',
      ...options,
    };
```